src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
That also needs to be changed to look like this:

```js
getRelevantContextKeys(muiTheme) {
    return {
      spacingDesktopGutterLess: muiTheme.rawTheme.spacing.desktopGutterLess,
    };
  },
```

and call it using `this.state.muiTheme` wherever it's called in the file. Please let me know if you find any more incorrect definitions of these functions in the source. Thanks!